framework/tester.py

```diff
@@ -298,12 +299,15 @@ def tearDown(self):
         self.__ips = []
 
         self.oops.update()
-        if self.oops._warn_count("Oops") > 0:
-            raise Exception("Oopses happened during test on Tempesta")
-        if self.oops._warn_count("WARNING") > 0:
-            raise Exception("Warnings happened during test on Tempesta")
-        if self.oops._warn_count("ERROR") > 0:
-            raise Exception("Errors happened during test on Tempesta")
+        for err in ["Oops", "WARNING", "ERROR"]:
+            if err in self.oops_ignore:
+                continue
+            if self.oops._warn_count(err) > 0:
+                self.oops_ignore = []
+                raise Exception("%s happened during test on Tempesta" % err)
```

Comment:
Don't we need to do the same for `testers/stress.py`?

Reply:
Yes, missed that for some reason. I've added the same interface for stress test. I will merge the PR as soon the tests on CI will pass